ci/do_ci.sh

```diff
@@ -382,6 +382,12 @@ elif [[ "$CI_TARGET" == "fix_spelling_pedantic" ]]; then
   echo "fix_spelling_pedantic..."
   ./tools/spelling/check_spelling_pedantic.py fix
   exit 0
+elif [[ "$CI_TARGET" == "headersplit" ]]; then
```

Comment:
I don't think this should have its own CI target, instead probably just insert it after https://github.com/envoyproxy/envoy/blob/master/ci/do_ci.sh#L250. I'd add a "TODO: consolidate this and the API tool tests in a dedicated target."